runtime/src/proposals_configuration/defaults.rs

```diff
@@ -347,3 +347,18 @@ pub(crate) fn unlock_blog_post_proposal() -> ProposalParameters<BlockNumber, Bal
         constitutionality: 1,
     }
 }
+
+// TODO: decide on paramaters
+// Proposal parameters for the 'Veto Proposal' proposal
+pub(crate) fn veto_proposal_proposal() -> ProposalParameters<BlockNumber, Balance> {
+    ProposalParameters {
```

Comment:
Please, add the data to the `sample_proposal_parameters.json` and add the `ignored` test for this proposal sample configuration.

Reply:
Do you mean [add this parameter to the json](https://github.com/Joystream/joystream/pull/2178/files#diff-289308fb6811411c14ee716c26aad5fa8b6a9376491ae692d64e4441fea1f715R242) and [this test](https://github.com/Joystream/joystream/pull/2178/files#diff-c03044c26ee60877126fa569ec9c78c6ed7bed8b1f280211cd7a4e3f070d1dc8R235) or am I missing something more?